code/modules/projectiles/projectile/beams.dm

```diff
@@ -40,8 +40,8 @@
 		impact_effect_type = /obj/effect/temp_visual/impact_effect/red_laser/wall
 
 /obj/projectile/beam/laser/lesslethal
-	damage = 10
-	stamina = 18
+	damage = 11
+	stamina = 25
```

Comment:
Stamian damage should be strictly lower than the stamina damage output of the disabler. All other things like increased ammo are fine, I was planning to put rechargers back in the heads offices.

Reply:
I'd rather the DPS be lower and they be given backup clips or the capability to recharge instead.

Reply:
One extra magazine has been added to each locker. I still don't like the idea of being able to create or order additional mags personally, but if it's limited to a single one for heads it should be fine.